### PR TITLE
Add news for SIGMOD Systems Award in 2022

### PR DESCRIPTION
**Pull request by HyukjinKwon**
This PR adds news for SIGMOD Systems Award in 2022, see also [SIGMOD Systems Award for Apache Spark](https://lists.apache.org/thread/1v0q2t7bc4zjt6v73r2pqxbtrq60oyyn) in Spark dev mailing list:

Screen Shot 2022-05-13 at 11 39 45 AM

In addition, this PR it adds GitHub blockquote CSS (that I copied from GitHub's).

### CODE REVIEW

File: css/custom.css [hunk 1]
```diff
@@ -964,3 +964,10 @@ h1 .tm, h2 .tm, h3 .tm {
     color: #555;
     font-weight: 500;
 }
+
+/* GitHub style blockquote */
```
Comment (HyukjinKwon) [inline css/custom.css:968]: cc @gengliangwang FYI

**Comment by HyukjinKwon** [comment]
Not sure my wording is good enough. cc @mateiz @dongjoon-hyun @srowen @gatorsmile @cloud-fan @gengliangwang FYI

**Review by srowen — APPROVED** [review]

File: news/_posts/2022-05-13-sigmod-system-award.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: SIGMOD System Award for Apache Spark
```
Comment (yaooqinn) [inline news/_posts/2022-05-13-sigmod-system-award.md]: Systems

File: news/_posts/2022-05-13-sigmod-system-award.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+tags: []
+status: publish
+type: post
+published: true
+meta:
+  _edit_last: '4'
+  _wpas_done_all: '1'
+---
+
+Apache Spark [received the SIGMOD System Award this year](https://sigmod.org/2022-sigmod-awards/),
```
Comment (yaooqinn) [inline news/_posts/2022-05-13-sigmod-system-award.md]: Systems

**Comment by yaooqinn** [comment]
`SIGMOD System Award` shall also be `SIGMOD Systems Award` in the title and desc of PR.

File: css/custom.css (hunk 1 above)
Comment (gengliangwang) [reply css/custom.css:968]: image

Let's try copying the "display: block; marigin-block-start..."

Comment (HyukjinKwon) [reply css/custom.css:968]: 🙆‍♂️

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM except @yaooqinn 's comment. Thank you, @HyukjinKwon .

**Comment by mateiz** [comment]
Looks good to me, but agree with @yaooqinn.

File: css/custom.css [hunk 4]
```diff
@@ -964,3 +964,19 @@ h1 .tm, h2 .tm, h3 .tm {
...
+blockquote {
+    display: block;
+    margin-block-start: 1em;
+    margin-block-end: 1em;
+    margin-inline-start: 40px;
+    margin-inline-end: 40px;
+}
+
+blockquote {
```
Comment (gengliangwang) [inline css/custom.css]: Merge this into line 975?

Comment (HyukjinKwon) [reply css/custom.css:977]: Merging it removes
image (1) for some reasons. Both are like that in GitHub CSS anyway. should be fine to leave it as is.

**Review by yaooqinn — APPROVED** [review]

**Review by gengliangwang — APPROVED** [review]

**Review by viirya — APPROVED** [review]

**Review by HeartSaVioR — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
Merged to asf-site.

**Pull request closed without merging**